src/test/kotlin/app/cash/licensee/LicenseePluginFixtureTest.kt

```diff
@@ -34,6 +34,7 @@ class LicenseePluginFixtureTest {
       "artifact-with-extension",
       "artifact-with-gradle-metadata-android",
       "artifact-with-gradle-metadata-jvm",
+      "artifact-with-scm",
```

Comment:
I would prefer if we removed this and instead added the cases to these three tests:
```
"pom-with-inheritance-from-both",
"pom-with-inheritance-from-child",
"pom-with-inheritance-from-parent",
```
This way we ensure that we accept the URL from the parent of not present in the child, and in the case when it's present in both we prefer the child.

Reply:
2cf332b